SortingMethods/bubbleSort.py

```diff
@@ -0,0 +1,17 @@
+def bubbleSort(numbersToSort):
+    Flag = True
+    while Flag:
+        Flag = False
```

Comment:
Ustawiasz flagę na true, po to by ją od razu odwrócić na false, po to by warunkowo z powrotem powrócić do true. Albo odwróć wartość flagi albo użyj while not, by uprościć zagmatwaną logikę.